src/Integration/MefServices/VsSessionHost.cs

```diff
@@ -323,7 +323,7 @@ private void RegisterLocalServices()
 
                 return new ProjectSystemFilter(this, testProjectIndicator);
             }));
-            this.localServices.Add(typeof(IErrorListInfoBarController), new Lazy<ILocalService>(() => new ErrorListInfoBarController(this, new UnboundProjectFinder(this, new ProjectBinderFactory()))));
+            this.localServices.Add(typeof(IErrorListInfoBarController), new Lazy<ILocalService>(() => new ErrorListInfoBarController(this, new UnboundProjectFinder(this, new ProjectBinderFactory(this)))));
```

Comment:
Looks like the creation of the ProjectBinderFactory could be moved to an internal constructor on both `RuleSetConflictsController` and `UnboundProjectFinder`. Doesn't need to be done now though if the code is still changing.